### PR TITLE
#322 JQL - snapshot version filter

### PR DESCRIPTION
**Pull request by mwesolowski**

### CODE REVIEW

**Comment by mwesolowski** [comment]
I don't know if it is not too late now but maybe we could think about wrapping snapshot version primitive into some object?

**Comment by bartoszwalacik** [comment]
imho long for snapshot.version is fine

File: javers-core/src/main/java/org/javers/repository/api/QueryParamsBuilder.java [hunk 1]
```diff
@@ -89,11 +90,19 @@ public QueryParamsBuilder commitId(CommitId commitId) {
         return this;
     }
 
+    /*
+     * limits results to Snapshots with a given version
+     */
+    public QueryParamsBuilder version(Long version) {
```
Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/repository/api/QueryParamsBuilder.java]: imo we should use `long` not `Long` here, the same as in `CdoSnapshot.getVersion()`

Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/repository/api/QueryParamsBuilder.java]: I will change it

Comment (bartoszwalacik) [inline javers-core/src/main/java/org/javers/repository/api/QueryParamsBuilder.java]: ok, int this context null means, dont use it in query

**Pull request merged**